.travis/build_doc.sh

```diff
@@ -2,5 +2,29 @@
 
 set -o errexit
 
+# PyQt >= 5.12 distributes WebEngine separately
+pip install pyqtwebengine
```

Comment:
This is not needed. Just remove the restriction on pyqt version in [conda_environment.yml](https://github.com/biolab/orange3-text/blob/629d9570ecec93b0e5d2a32c19959a9bd91b0343/conda_environment.yml#L15). The import qtwebengine error resolves itself.